test/integration/targets/iosxr_acl_interfaces/tests/cli/_populate.yaml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Populate the device with ACLs
+  iosxr_config:
+    lines: |
+      ipv4 access-list acl_1
+      10 permit ipv4 any any
+      ipv4 access-list acl_2
+      10 permit ipv4 any any
+      ipv4 access-list acl_3
+      10 permit ipv4 any any
+      ipv6 access-list acl6_1
+      10 permit ipv6 any any
+      ipv6 access-list acl6_2
+      10 permit ipv6 any any
+      ipv6 access-list acl6_3
+      10 permit ipv6 any any
+
+- name: Setup ACL interfaces configuration for GigabitEthernet0/0/0/0
+  iosxr_config:
```

Comment:
Can iosxr_acl_interfaces be used instead of iosxr_config ?

Reply:
I feel that Setup/Remove tasks shouldn't use the module that is being tested.